src/Controller/Helper/ProgrammeRequestContentType.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller\Helper;
+
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class ProgrammeRequestContentType
+{
+    public function getRequestType(Request $request): string
+    {
+        $acceptHeader = $request->headers->get('accept');
+        $mimeTypes = explode('/', $acceptHeader);
+
+        return $mimeTypes[1];
```

Comment:
what if the header is missing?  what if the headers is not in the format "application/something"?